agent-tools/poll_reviews.py

```
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+# Can be overridden by CLI args
+LAST_REVIEW_DATE = sys.argv[1] if len(sys.argv) > 1 else "2026-01-02T00:17:24Z"
+PR_NUMBER = 29
+TIMEOUT = 1200  # 20 minutes
+
+def fetch_reviews(temp_dir):
+    output_path = os.path.join(temp_dir, "reviews_temp.json")
+    try:
+        # Fetch reviews
+        subprocess.run(
```

Ruff (0.14.10)

`subprocess` call with `shell=True` identified, security issue

(S602)